ci/generate_azure_pipelines_matrices.py

```diff
@@ -95,20 +104,21 @@ def __init__(self, platform, build_jobs):
     [
         BuildJob(
             "install-qt",
-            "5.14.2",
+            "5.15.2",
             "windows",
             "desktop",
-            "win64_msvc2017_64",
-            "msvc2017_64",
+            "win32_msvc2019",
+            "msvc2019",
```

Comment:
Qt 5.14.2 has 6 architectures available: `win32_mingw73`, ~~`win32_msvc2017`~~, `win64_mingw73`, `win64_msvc2015_64`, ~~`win64_msvc2017_64`~~, and `wasm_32`. On the Windows-2019 host, MSVC 2017 is no longer available. I have updated the other MSVC 2017 builds to use the other mingw architectures. At this point, all of the architectures available for Qt 5.14.2 are being tested. I didn't know what else to do for this build, so I added the only architecture that wasn't being tested for MSVC 2019, and I changed this build to Qt 5.16.2.

If you disagree with any of the build jobs being tested here, please suggest alternatives and I will be happy to test them.